app/views/admin/users/_search_form_for.html.slim

```diff
@@ -0,0 +1,5 @@
+= search_form_for @search, url: admin_users_path do |f|
+  .field
+    = f.label :first_name_cont, "First Name contains"
```

Comment:
try to doit with bootstrap
<div class="col-lg-6">
    <div class="input-group">
      <input type="text" class="form-control" placeholder="Search for...">
      <span class="input-group-btn">
        <button class="btn btn-default" type="button">Go!</button>
      </span>
    </div>
  </div>